main.go

```diff
@@ -148,6 +149,7 @@ func registerCapabilities(port string, log logger.Handler) {
 	if err := oam.RegisterMeshModelComponents(instanceID, mesheryServerAddress(), serviceAddress(), port); err != nil {
 		log.Info(err.Error())
 	}
+	log.Info("Successfully registered static components with Meshery Server.")
```

Comment:
```suggestion
	log.Info("Registered static components with Meshery Server.")
```

It would be great if we could include a count of the number of components registered.